samples/snippets/csharp/api/system.reflection.metadata.ecma335/metadatabuilder/Program.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace MetadataBuilderSnippets
+{
+    class Program
+    {
+        static void Main(string[] args)

```

Comment:
This file is not added to the example, please add or remove the file

Reply:
It was meant as a standard boilerplate entrypoint to make snippets runnable in CI in the future. But it's actually not needed here as i only have one snippet file, so, i removed it.